cmd/proxy/actions/dotfile.go

```diff
@@ -0,0 +1,30 @@
+package actions
+
+import (
+	"io/ioutil"
+	"log"
+	"path/filepath"
+
+	"github.com/mitchellh/go-homedir"
+)
+
+// initializeDotFile checks if provided dot file is at a pre-configured path
+// and moves to home directory -- note that this will override whatever
+// dot file you have in your home directory.
+func initializeDotFile(path string) {
+	if path == "" {
+		return
+	}
+
+	fileBts, err := ioutil.ReadFile(path)
+	if err != nil {
+		log.Fatalf("could not read %s: %v", path, err)
+	}
+
+	hdir, err := homedir.Dir()
+	if err != nil {
+		log.Fatalf("could not get homedir: %v", err)
+	}
+	rcp := filepath.Join(hdir, filepath.Base(path))
+	ioutil.WriteFile(rcp, fileBts, 0666)
```

Comment:
Is it a hard req to have 0666 as permission, anyone from the world can read it.
I think the best is 0600 or 0660. Can you check what is recommended by hgrc or change the file permission.

Reply:
It seems like previously for .netrc file got to much permission (https://www.labkey.org/Documentation/wiki-page.view?name=netrc), will fix that after investigating minimal permission for .hgrc